mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
+      if (!deletedOrder) {
+        return res.status(404).json({ error: "No such order found" });
+      }
+      res.json({ success: "Order has been deleted" });
```

Comment:
The success message should be consistent throughout the API. Instead of hardcoding the success message, consider defining a constant for the success message in a separate file, e.g., `constants.js`, and import it here. This will ensure consistency in the success messages across the API.